varcode/vcf.py

```diff
@@ -296,6 +301,9 @@ def dataframes_to_variant_collection(
 
     variant_collection_kwargs : dict, optional
         Additional keyword parameters to pass to VariantCollection.__init__.
+
+    sources : set or list
```

Comment:
I'd say it's useful? A common operation for us in `cohorts` is just saving/loading the `EffectCollection` and not always having the `VariantCollection` around. I think they should be able to stand on their own, but obviously if you take it out we can reference the variant collection